tests/ganache.go

```diff
@@ -1,3 +1,4 @@
+// Package tests provides
```

Comment:
complete comment?

Reply:
Ops. I should have deleted that. I was experimenting with moving this file to a different package, but decided against it. The package comments are already in integration_test.go.